src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -565,8 +565,9 @@ private bool TryGetMentorNode(string dbName, DatabaseTopology topology, ClusterT
             {
                 if (lastSentEtag < mentorsEtag)
                 {
-                    var msg = $"The database {dbName} on {promotable} not ready to be promoted, because the mentor hasn't sent all his documents.\n" +
-                              $"Last sent Etag: {lastSentEtag}, Mentor's Etag: {mentorsEtag}";
+                    var msg = $"The database {dbName} on {promotable} not ready to be promoted, because the mentor hasn't sent all of the documents yet." + Environment.NewLine +
```

Comment:
Change to: 
The database '{dbName}' on node {promotable} is not ready to be promoted, because the mentor hasn't sent all of the documents yet.